core/arch/riscv/plat-virt/sub.mk

```diff
@@ -0,0 +1,2 @@
+global-incdirs-y += .
```

Comment:
Unlike what existing platforms did, I would suggest to have a dedicated `include/` subdir for header files to be exposed the the whole OP-TEE core. FWIW.

Reply:
Move platform_config.h to include folder ?

Reply:
I would do that for a new platform. When plat-foo/ gets populated with more sources files it may be not handy to have local header files exposed to the whole core sources.
All arm platforms do as you did and they don't have problems. It's really a suggestion. Maybe addressed later, when needed.